frontend/src/index.tsx

```diff
@@ -17,6 +17,7 @@ import { Auth } from '#pages/auth/auth.js';
 import { Chat } from '#pages/chat/chat.js';
 import { Dashboard } from '#pages/dashboard/dashboard.js';
 import { Journal } from '#pages/journal/journal.js';
+import { MeditationPlayer } from '#pages/meditation/components/meditation-player/meditation-player.js';
```

Comment:
```suggestion
import { MeditationPlayer } from '#pages/meditation/meditation.js';
```

Reply:
There is no file with re-exports in meditation folder, the only meditation.js is the parent component that renders sidebar, list and player